R/MeasureDensSquaredloss.R

```diff
@@ -39,38 +39,42 @@ MeasureDensSquaredloss = R6::R6Class("MeasureDensSquaredloss",
 
         private = list(
         .eps = numeric(0),
-        .score = function(prediction, learner, task, train_set, ...) {
+        .score = function(prediction, learner, task, train_set) {
          # return NA if learner not compatible
          # change `c("dens.kde")` to list of compatible learners
-           data = task$truth(train_set)
 
-           bw = learner$train(task)$model$bw
-           kernel = learner$train(task)$model$kernel
 
-          # vectorized the data
-          dat <- sapply(data, function (x, y) ((x - y) / bw), y = data)
+           # bw = learner$train(task)$model$bw
+           # kernel = learner$train(task)$model$kernel
```

Comment:
Is this just temporary code? The learner should never be re-trained in the score

Reply:
I removed this